nflow-engine/src/main/java/io/nflow/engine/internal/dao/WorkflowInstanceDao.java

```diff
@@ -99,8 +98,8 @@
   private long workflowInstanceQueryMaxResultsDefault;
   private long workflowInstanceQueryMaxActions;
   private long workflowInstanceQueryMaxActionsDefault;
-  int instanceStateTextLength;
-  int actionStateTextLength;
+  int instanceStateTextLength = -1;
```

Comment:
No other than those didn't have private before, so didn't think to check. Added privates in upcoming commit.